lib/membrane/video_compositor/wgpu/wgpu.ex

```diff
@@ -2,18 +2,30 @@ defmodule Membrane.VideoCompositor.Wgpu do
   @moduledoc """
   This module implements video composition in wgpu
   """
-  @behaviour Membrane.VideoCompositor.FrameCompositor
 
-  alias Membrane.VideoCompositor.Common.{RawVideo, VideoProperties}
+  alias Membrane.VideoCompositor.Common
```

Comment:
I think you could change the docs in Common, because now there is only one implementation

Reply:
Yep, good point. I should also rename `Common`, since we don't use it in multiple implementations.

Reply:
Fixed. I have moved it to `Membrane.VideoCompositor.RustStructs.RawVideo` and `Membrane.VideoCompositor.RustStructs.VideoProperties`